docs/6.x/pack.md

```diff
@@ -465,13 +465,16 @@ systemOptions:
     args: ["--system-reserved=memory=500Mi"]
     hairpinMode: "promiscuous-bridge"
 
+  # When set to true, allows running privileged containers in the deployed
+  # clusters, defaults to false
+  allowPrivileged: true
```

Comment:
It's hard to see the context here, but do we want to set this true? If it's a default example?

Reply:
Well, the purpose of this particular sample manifest is to show how to enable everything you can possibly enable. So this is not technically "default" manifest you'd want to use. I'm okay with setting this to false here though.

Reply:
I think if you look at the way we do other samples, such as extensions.logs.disabled we show the default value (false). So it would seem to match to have allowPrivileged show the default value as well.

Reply:
Yeah, I'll change it to false.